packages/manager/modules/hub/src/components/billing-summary/controller.js

```diff
@@ -19,6 +20,9 @@ export default class ManagerHubBillingSummaryCtrl {
           get(data, 'currency.code'),
         );
         return this.bills;
+      })
+      .finally(() => {
+        this.loading = false;
```

Comment:
`loading` only concerns bills loading ? Not loading debt also ?

Reply:
You're right, loading was not put at the the right place